File: remote/src/androidTest/java/com/kelsos/mbrc/MainActivityTest.java
package com.kelsos.mbrc;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.annotation.Config;

import static junit.framework.Assert.assertTrue;

@Config(emulateSdk = 18) @RunWith(com.kelsos.mbrc.RobolectricGradleTestRunner.class)
public class MainActivityTest {

  @Test public void shouldPass() {
    assertTrue("is true", true);
  }
}

File: app/src/androidTest/kotlin/com/kelsos/mbrc/MainActivityTest.kt
package com.kelsos.mbrc

import junit.framework.Assert.assertTrue
import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.annotation.Config

@Config(emulateSdk = 18)
@RunWith(com.kelsos.mbrc.RobolectricGradleTestRunner::class)
class MainActivityTest {

    @Test fun shouldPass() {
        assertTrue("is true", true)
    }
}
